source/administration/command-line-tools.rst

```diff
@@ -0,0 +1,875 @@
+Command Line Tools
+==================
+
+From the directory where the Mattermost platform is installed, a
+``platform`` command is available for configuring the system, including:
+
+**General Administration**
+
+-  Creating teams
+-  Creating users
+-  Assigning roles to users
+-  Reseting user passwords
+-  Inviting users to teams
+
+**Advanced Administration**
+
+-  Permanently deleting users (use cautiously - database backup
+   recommended before use)
+-  Permanently deleting teams (use cautiously - database backup
+   recommended before use)
+
+**Advanced Automation**
+
+*Available in Enterprise Edition E10 and higher*
+
+-  Creating channels
+-  Inviting users to channels
+-  Removing users from channels
+-  Listing all public channels and private groups for a team
+-  Restoring previously deleted channels
+-  Migrating sign-in options
+-  Resetting multi-factor authentication for a user
+
+CLI Commands
+------------
+
+Typing ``platform gendocs`` returns documentation for the CLI tool.
+
+Notes:
+
+-  Parameters in CLI commands are order-specific.
+-  If special characters (``!``, ``|``, ``(``, ``)``, ``\``,
+   \`\`\`\ ``, and``"``) are used, the entire argument needs to be surrounded by single quotes (e.g.``-password
+   'mypassword!'``, or the individual characters need to be escaped out (e.g.``-password
+   mypassword/!\`\`).
+-  Team name and channel name refer to the handles, not the display
+   names. So in the url
+   ``https://pre-release.mattermost.com/core/channels/town-square`` team
+   name would be ``core`` and channel name would be ``town-square``
+
+.. tip :: If you automate user creation through the CLI tool with SMTP
+enabled emails will be sent to all new users created. If you run such a
+load script, it is best to disable SMTP or to use test accounts so that
+new account creation emails aren't unintentionally set to people at your
+organization who aren't expecting them.
+
+platform
+~~~~~~~~
+Commands for configuring and managing your Mattermost instance and users.
+
+**Options**
+
+::
+
+      -c, --config string   Configuration file to use. (default "config.json")
+
+**Child Commands**
+
+-  `platform channel`_ - Management of channels
+-  `platform gendocs`_ - Generate full documentation for the CLI
+-  `platform import`_ - Import data
+-  `platform ldap`_ - LDAP related utilities
+-  `platform license`_ - Licensing commands
+-  `platform reset`_ - Reset the database to initial state
+-  `platform roles`_ - Management of user roles
+-  `platform server`_ - Run the Mattermost server
+-  `platform team`_ - Management of teams
+-  `platform user`_ - Management of users
+-  `platform version`_ - Display version information
+
+platform channel
+~~~~~~~~~~~~~~~~
+
+Commands for channel management.
+
+**Options inherited from parent commands**
```

Comment:
Can we remove all the "Options inherited from parent commands." Its unnecessary to repeat